docs/source/ref/templatetags.rst

```diff
@@ -159,6 +159,8 @@ or
 
     {% oscar_thumbnail [source] [size] [options] as [variable] %}
 
+When the `OSCAR_THUMBNAIL_DEBUG` setting is set to True, this templatetag will fail with an error if an exception is raised while generating the thumbnail. If set to False, an empty string is returned.
```

Comment:
Please fix rendering - variables including things like `True` and `False` need to be in double backticks, not single. Also "template tag" should be two words.